cityofaustin/atd-vz-data
# Patch missing near trauma fields and revoke Editor access to Changes tab

## Associated issues
This is a patch to get PR https://github.com/cityofaustin/atd-vz-data/pull/1348 into production. Adds the new near trauma fields from CRIS to the crashes table and removes Editor users from being able to access the Changes tab.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch
2. Get a fresh copy of VZ prod data running locally
3. Run the up/down migrations
4. Folllow other test steps in original PR

---
#### Ship list
